.github/workflows/test-and-build.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         with:
           path: ./symfem
           repository: mscroggs/symfem
-          ref: main
```

Comment:
This needs resetting to main once https://github.com/mscroggs/symfem/pull/293 is merged